### PR TITLE
yasnippet: theme yas-snippet-dirs

### PR DESCRIPTION
**Pull request by benjaminfrank**
Theme `yas-snippet-dirs' and create required directory.

### CODE REVIEW

**Comment by benjaminfrank** [comment]
I kept the defautl yasnippet foldername ('snippets/'). If I should change that according to the rules (probably should be 'yasnippet/' then) I can easily change the PR.

File: no-littering.el [hunk 1]
```diff
@@ -238,7 +238,9 @@ This variable has to be set before `no-littering' is loaded.")
     (setq sx-cache-directory               (var "sx-cache/"))
     (setq undo-tree-history-directory-alist (list (cons "." (var "undo-tree-hist/"))))
     (setq user-emacs-ensime-directory      (var "ensime/"))
-    ))
```
Comment (tarsius) [inline no-littering.el]: Please keep that line as-is.

**Comment by tarsius** [comment]
I think the `yas` should be in there, so lets use `yas-snippets/`.

**Comment by benjaminfrank** [comment]
done.

**Comment by tarsius** [comment]
Thanks!

**Pull request merged**